### ISSUE TITLE
java.lang.NullPointerException

### ISSUE THREAD

**Issue by Sonlein** [Sashie/skript-yaml#23]
```
[00:20:36] [Server thread/ERROR]: #!#! java.lang.NullPointerException
[00:20:36] [Server thread/ERROR]: #!#!     at me.sashie.skriptyaml.skript.ExprYaml.parseString(ExprYaml.java:293)
[00:20:36] [Server thread/ERROR]: #!#!     at me.sashie.skriptyaml.skript.ExprYaml.arrayToList(ExprYaml.java:288)
[00:20:36] [Server thread/ERROR]: #!#!     at me.sashie.skriptyaml.skript.ExprYaml.change(ExprYaml.java:280)
[00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.effects.EffChange.execute(EffChange.java:282)
[00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.Effect.run(Effect.java:51)
[00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:60)
[00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:88)
[00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.Trigger.execute(Trigger.java:56)
[00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.events.EvtScript.register(EvtScript.java:60)
[00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader$4.call(ScriptLoader.java:737)
[00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader$4.call(ScriptLoader.java:1)
[00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader.loadScript(ScriptLoader.java:760)
[00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader.lambda$4(ScriptLoader.java:401)
[00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader.loadScripts(ScriptLoader.java:420)
[00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader.lambda$3(ScriptLoader.java:337)
[00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader.loadScripts(ScriptLoader.java:370)
[00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.Skript.reloadScripts(Skript.java:999)
[00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.SkriptCommand.onCommand(SkriptCommand.java:165)
[00:20:36] [Server thread/ERROR]: #!#!     at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
[00:20:36] [Server thread/ERROR]: #!#!     at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:149)
[00:20:36] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_16_R1.CraftServer.dispatchCommand(CraftServer.java:757)
[00:20:36] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_16_R1.CraftServer.dispatchServerCommand(CraftServer.java:742)
[00:20:36] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R1.DedicatedServer.handleCommandQueue(DedicatedServer.java:386)
[00:20:36] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R1.DedicatedServer.b(DedicatedServer.java:355)
[00:20:36] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R1.MinecraftServer.a(MinecraftServer.java:1009)
[00:20:36] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:848)
[00:20:36] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R1.MinecraftServer.lambda$0(MinecraftServer.java:164)
[00:20:36] [Server thread/ERROR]: #!#!     at java.base/java.lang.Thread.run(Unknown Source)
[00:20:36] [Server thread/ERROR]: #!#!
[00:20:36] [Server thread/ERROR]: #!#! Version Information:
[00:20:36] [Server thread/ERROR]: #!#!   Skript: 2.5.1 (latest)
[00:20:36] [Server thread/ERROR]: #!#!     Flavor: skriptlang-github
[00:20:36] [Server thread/ERROR]: #!#!     Date: 2020-10-22
[00:20:36] [Server thread/ERROR]: #!#!   Bukkit: 1.16.1-R0.1-SNAPSHOT
[00:20:36] [Server thread/ERROR]: #!#!   Minecraft: 1.16.1
[00:20:36] [Server thread/ERROR]: #!#!   Java: 9.0.4 (Java HotSpot(TM) 64-Bit Server VM 9.0.4+11)
[00:20:36] [Server thread/ERROR]: #!#!   OS: Windows 10 amd64 10.0
[00:20:36] [Server thread/ERROR]: #!#!
[00:20:36] [Server thread/ERROR]: #!#! Server platform: Spigot
[00:20:36] [Server thread/ERROR]: #!#!
[00:20:36] [Server thread/ERROR]: #!#! Current node: null
[00:20:36] [Server thread/ERROR]: #!#! Current item: set yaml list "test.result" from "smithing" to {_a::*}(as java.lang.Object)
```

skript code:
```
on script load:
  load yaml "plugins/RPGProfessions/smithing.yml" as "smithing"
  set {_a::1::chance} to 20
  set {_a::1::item} to "failed_iron_ingot"
  set {_a::2::chance} to 50
  set {_a::2::item} to "forged_iron_ingot"
  set {_a::3::chance} to 20
  set {_a::3::item} to "good_iron_ingot"
  set {_a::4::chance} to 10
  set {_a::4::item} to "rare_iron_ingot"
  set yaml list "test.result" from "smithing" to {_a::*}
  save yaml "smithing"
```

**Comment by Olyno** [comment]
What version of skript-yaml are you using?

**Comment by Sonlein** [comment]
> What version of skript-yaml are you using?

1.3.2

**Comment by wallace4BR** [comment]
> ```
> [00:20:36] [Server thread/ERROR]: #!#! java.lang.NullPointerException
> [00:20:36] [Server thread/ERROR]: #!#!     at me.sashie.skriptyaml.skript.ExprYaml.parseString(ExprYaml.java:293)
> [00:20:36] [Server thread/ERROR]: #!#!     at me.sashie.skriptyaml.skript.ExprYaml.arrayToList(ExprYaml.java:288)
> [00:20:36] [Server thread/ERROR]: #!#!     at me.sashie.skriptyaml.skript.ExprYaml.change(ExprYaml.java:280)
> [00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.effects.EffChange.execute(EffChange.java:282)
> [00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.Effect.run(Effect.java:51)
> [00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:60)
> [00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:88)
> [00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.Trigger.execute(Trigger.java:56)
> [00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.events.EvtScript.register(EvtScript.java:60)
> [00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader$4.call(ScriptLoader.java:737)
> [00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader$4.call(ScriptLoader.java:1)
> [00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader.loadScript(ScriptLoader.java:760)
> [00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader.lambda$4(ScriptLoader.java:401)
> [00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader.loadScripts(ScriptLoader.java:420)
> [00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader.lambda$3(ScriptLoader.java:337)
> [00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.ScriptLoader.loadScripts(ScriptLoader.java:370)
> [00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.Skript.reloadScripts(Skript.java:999)
> [00:20:36] [Server thread/ERROR]: #!#!     at ch.njol.skript.SkriptCommand.onCommand(SkriptCommand.java:165)
> [00:20:36] [Server thread/ERROR]: #!#!     at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
> [00:20:36] [Server thread/ERROR]: #!#!     at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:149)
> [00:20:36] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_16_R1.CraftServer.dispatchCommand(CraftServer.java:757)
> [00:20:36] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_16_R1.CraftServer.dispatchServerCommand(CraftServer.java:742)
> [00:20:36] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R1.DedicatedServer.handleCommandQueue(DedicatedServer.java:386)
> [00:20:36] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R1.DedicatedServer.b(DedicatedServer.java:355)
> [00:20:36] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R1.MinecraftServer.a(MinecraftServer.java:1009)
> [00:20:36] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:848)
> [00:20:36] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R1.MinecraftServer.lambda$0(MinecraftServer.java:164)
> [00:20:36] [Server thread/ERROR]: #!#!     at java.base/java.lang.Thread.run(Unknown Source)
> [00:20:36] [Server thread/ERROR]: #!#!
> [00:20:36] [Server thread/ERROR]: #!#! Version Information:
> [00:20:36] [Server thread/ERROR]: #!#!   Skript: 2.5.1 (latest)
> [00:20:36] [Server thread/ERROR]: #!#!     Flavor: skriptlang-github
> [00:20:36] [Server thread/ERROR]: #!#!     Date: 2020-10-22
> [00:20:36] [Server thread/ERROR]: #!#!   Bukkit: 1.16.1-R0.1-SNAPSHOT
> [00:20:36] [Server thread/ERROR]: #!#!   Minecraft: 1.16.1
> [00:20:36] [Server thread/ERROR]: #!#!   Java: 9.0.4 (Java HotSpot(TM) 64-Bit Server VM 9.0.4+11)
> [00:20:36] [Server thread/ERROR]: #!#!   OS: Windows 10 amd64 10.0
> [00:20:36] [Server thread/ERROR]: #!#!
> [00:20:36] [Server thread/ERROR]: #!#! Server platform: Spigot
> [00:20:36] [Server thread/ERROR]: #!#!
> [00:20:36] [Server thread/ERROR]: #!#! Current node: null
> [00:20:36] [Server thread/ERROR]: #!#! Current item: set yaml list "test.result" from "smithing" to {_a::*}(as java.lang.Object)
> ```
> 
> skript code:
> 
> ```
> on script load:
>   load yaml "plugins/RPGProfessions/smithing.yml" as "smithing"
>   set {_a::1::chance} to 20
>   set {_a::1::item} to "failed_iron_ingot"
>   set {_a::2::chance} to 50
>   set {_a::2::item} to "forged_iron_ingot"
>   set {_a::3::chance} to 20
>   set {_a::3::item} to "good_iron_ingot"
>   set {_a::4::chance} to 10
>   set {_a::4::item} to "rare_iron_ingot"
>   set yaml list "test.result" from "smithing" to {_a::*}
>   save yaml "smithing"
> ```
This issue was caused by the way you used the list variables
you didn't set any value to {_a::*} only for {_a::1::*} and so
broadcast "%{_a::1::*}%" This will work but you won't get all values that you want
broadcast "%{_a::*}%" This will return <none>

**Comment by wallace4BR** [comment]
Btw, the error caused by the variable was fixed in 1.4

**Comment by Sashie** [comment]
<3
